lib/factory/EntryFieldDescription.js

```diff
@@ -5,45 +5,69 @@ var escapeHTML = require('../Utils').escapeHTML;
 /**
  * Create a linkified and HTML escaped entry field description.
  *
- * As a special feature, this description may contain both markdown
- * and plain <a href> links.
+ * As a special feature, this description may contain both markdown,
+ * plain <a href> links and <br />
  *
  * @param {String} description
  */
 module.exports = function entryFieldDescription(description) {
 
   // we tokenize the description to extract text, HTML and markdown links
-  // text and links are handled seperately
+  // text, links and new lines are handled seperately
 
   var escaped = [];
 
   // match markdown [{TEXT}]({URL}) and HTML links <a href="{URL}">{TEXT}</a>
-  var pattern = /(?:\[([^\]]+)\]\((https?:\/\/[^"<>\]]+)\))|(?:<a href="(https?:\/\/[^"<>]+)">([^<]*)<\/a>)/gi;
+  var urlPattern = /(?:\[([^\]]+)\]\((https?:\/\/[^"<>\]]+)\))|(?:<a href="(https?:\/\/[^"<>]+)">(.+?(?=<\/))<\/a>)/gi;
```

Comment:
Changed here in order to handle line breaks inside text fields of <a> tag.

[^<]*  accepts strings until < is seen, which does not go well for something like: <a href="..."> hello <br/> world </a>

So I changed this to -> .+?(?=<\/) which accepts strings until </ is seen.